api/models.py

```diff
@@ -136,10 +140,15 @@ class ReportType(models.TextChoices):
         on_delete=models.SET_NULL,
         limit_choices_to={"is_staff": True},
         null=True,
+        blank=True,
         help_text="The admin handling the report",
         related_name="assigned_reports",
     )
     video = models.ForeignKey(
```

Comment:
Was this the change from master or you made this change?

Reply:
No. Github is a bit weird when showing the changes from a merge. Click on the latest commit hash to see the changes I made to fix the merge conflicts.

Reply:
The change was made in this PR but not to fix the merge.